companion/src/firmwares/opentx/opentxinterface.cpp

```diff
@@ -543,6 +543,13 @@ int OpenTxFirmware::getCapability(::Capability capability)
         return 0;
     case MaxVolume:
       return (IS_ARM(board) ? 23 : 7);
+    case MaxContrast:
+      if (IS_TARANIS_X7(board)) return 30;
```

Comment:
Line break after `if` please (also below).

Reply:
why? the `return 45;` does the job